src/Uno.Templates/content/unoapp/Directory.Build.props

```diff
@@ -62,6 +68,7 @@
 				<SupportedOSPlatformVersion>10.0.18362.0</SupportedOSPlatformVersion>
 				<TargetPlatformMinVersion>10.0.18362.0</TargetPlatformMinVersion>
 				<RuntimeIdentifiers>win10-x86;win10-x64;win10-arm64</RuntimeIdentifiers>
+				<EnableCoreMrtTooling Condition=" '$(BuildingInsideVisualStudio)' != 'true' ">false</EnableCoreMrtTooling>
```

Comment:
This is to allow `dotnet build` to work for WinUI from command-line?

Would like @jeromelaban to confirm if there are any concerns with this.

Reply:
Correct, that's what this is for. Otherwise have to use msbuild